File: act.java
// Function to compute the activities to be chosen
    public static void activities(int start[], int finish[], int n)
    {
        int i = 0, j;
        // Select the first activity
        System.out.print( i + " ");
        // if start time of current activity j is greater than or equal to previous activity chosen, select activity j
        for (j = 1; j < n; j++)
        {
            if (start[j] >= finish[i])
            {
                System.out.print( j + " ");
                i = j;
            }
        }
    }

File: act.rb
n = start.length()
def print_activities(start , finish , n)
  print "Following activities are selected \n "
  i = 0
  print (i.to_s + "\t")
  for j in 1..n-1
    if start[j]>finish[i]
      print (j.to_s + "\t")
      i = j
    end
  end
end
